app/views/users/show.html.erb

```diff
@@ -1 +1,14 @@
-<h1>User Profile #<%= @id %></h1>
\ No newline at end of file
+<section>
+    <%= render "shared/user_card", user: @user %>
+
+    <div class="user-bio">
```

Comment:
Thanks for your suggestion. I'll make the required changes.

Reply:
Hello @IndieCoderMM  , thanks for your questions but it seems you have some troubles according to some request changes , I can assist you and explain more about it , let have a zoom call and I will try to explain everything :wink:

Reply:
Hi @Alfredbis29, 

I updated model validations to allow `nil` values so that models can be created without passing counters as you suggested. 

Thanks so much for providing me with helpful feedback and suggestions.  😃